packages/babel-parser/test/fixtures/experimental/destructuring-private/valid-arrow-params/output.json

```diff
@@ -1,9 +1,6 @@
 {
   "type": "File",
   "start":0,"end":46,"loc":{"start":{"line":1,"column":0},"end":{"line":4,"column":1}},
-  "errors": [
-    "SyntaxError: Private names can only be used as the name of a class element (i.e. class C { #p = 42; #m() {} } )\n or a property of member expression (i.e. this.#p). (3:11)"
-  ],
```

Comment:
This test has been moved to `packages/babel-parser/test/fixtures/experimental/_no-plugin/destructuring-private-arrow-params/input.js` and throws expected plugins instead.

This test is also duplicated to `packages/babel-parser/test/fixtures/experimental/destructuring-private/valid-arrow-params/input.js` as a test case for destructuring private. Here git confusingly marks that `output.json` is moved to `destructuring-private`.